scripts/install_environment.sh

```diff
@@ -183,6 +183,10 @@ function additional_configs() {
     sudo setenforce 0 || true
     sudo chmod 600 ssh_key/key
     sudo firewall-cmd --zone=libvirt --add-port=59151-59154/tcp
+
+    echo "enabling ipv6"
+    sudo sed -ir 's/net.ipv6.conf.all.disable_ipv6 = 1/net.ipv6.conf.all.disable_ipv6 = 0/g' /etc/sysctl.conf
```

Comment:
Although having spaces around the equal sign is the default, the format without spaces is also accepted. I'd recommend handling it as well, i.e. `net.ipv6.conf.all.disable_ipv6=1`. Just in case.

Reply:
Done